web/themes/custom/novel/templates/paragraphs/paragraph--accordion.html.twig

```diff
@@ -0,0 +1,11 @@
+<details class="disclosure disclosure--paragraph-width text-body-large">
```

Comment:
please remember to use ``{{ attributes.addClass(....) }}`, so we can also control it through future `preprocess`'es

Reply:
Good point, will do!